inc/ticket.class.php

```diff
@@ -4396,7 +4395,9 @@ function showForm($ID, $options = []) {
          $options['content'] = str_replace($order, $replace, $options['content']);
       }
       if (isset($options['name'])) {
-         $options['name'] = str_replace("\\\\", "\\", $options['name']);
+         $order           = ["\\\\\\'", '\\\\\\"', "\\\\\\\\"];
```

Comment:
Huh... really? I know dealing with `\` stuff is not something easy on GLPI... But we cannot add backslashes again and again

Reply:
I'm seeing more details